lib/gollum/public/gollum/stylesheets/template.scss.erb

```diff
@@ -199,311 +173,6 @@ img {
-    border-bottom: 1px solid #eee;
-  }
-
-  p, blockquote, ul, ol, dl, table, pre {
-    margin: 15px 0;
-  }
-
-  ul, ol {
-    padding-left: 30px;
-  }
-
-  dl {
-    padding: 0;
-
-    dt {
-      font-size: 14px;
-      font-weight: bold;
-      font-style: italic;
-      padding: 0;
-      margin: 15px 0 5px;
-
-      &:first-child {
-        padding: 0;
-      }
-
-      @include children-truncate-margin;
-    }
-
-    dd {
-      margin: 0 0 15px;
-      padding: 0 15px;
-
-      @include children-truncate-margin;
-    }
-  }
-
-  li {
-    margin: 0px;
-
-    p.first {
-      display: inline-block;
-    }
-  }
-
-  hr {
-  }
-
-  blockquote {
-    border-left: 4px solid #DDD;
-    padding: 0 15px;
-    color: #777;
-
-    @include children-truncate-margin;
-  }
-
-  table {
-    border-collapse: collapse;
-    border: $border-standard;
-    border-radius: 3px !important;
-    
-    tr {
-      border-top: 0;
-    }
-    tr th {
-      border-top: 0;
-    }
-    th:first-child {
-      border-left: 0;
-    }
-    th:last-child {
-      border-right: 0;
-    }
-    tr:first-child td {
-      border-top: 0;
-    }
-    tr td:first-child {
-      border-left: 0;
-    }
-    tr:last-child td {
-      border-bottom: 0;
-    }
-    tr td:last-child {
-      border-right: 0;
-    }
-    tfoot {
-      border-top: $border-standard;
-      th {
-        border-bottom: 0;
-      }
-    }
-  }
-  
-  img {
-    max-width: 100%;
-  }
-
-  span.frame {
-    display: block;
-    overflow: hidden;
-
-    > span {
-      border: $border-standard;
-      display: block;
-      float: left;
-      overflow: hidden;
-      margin: 13px 0 0;
-      padding: 7px;
-      width: auto;
-    }
-
-    span {
-      img {
-        display: block;
-        float: left;
-      }
-
-      span {
-        clear: both;
-        color: #333;
-        display: block;
-        padding: 5px 0 0;
-      }
-    }
-  }
-
-  span.align-center {
-    display: block;
-    overflow: hidden;
-    clear: both;
-
-    > span {
-      display: block;
-      overflow: hidden;
-      margin: 13px auto 0;
-      text-align: center;
-    }
-
-    span {
-      img {
-        margin: 0 auto;
-        text-align: center;
-      }
-    }
-  }
-
-  span.align-right {
-    display: block;
-    overflow: hidden;
-    clear: both;
-
-    > span {
-      display: block;
-      overflow: hidden;
-      margin: 13px 0 0;
-      text-align: right;
-    }
-
-    span {
-      img {
-        margin: 0;
-        text-align: right;
-      }
-    }
-  }
-
-  span.float-left {
-    display: block;
-    margin-right: 13px;
-    overflow: hidden;
-    float: left;
-
-    span {
-      margin: 13px 0 0;
-    }
-  }
-
-  span.float-right {
-    display: block;
-    margin-left: 13px;
-    overflow: hidden;
-    float: right;
-
-    > span {
-      display: block;
-      overflow: hidden;
-      margin: 13px auto 0;
-      text-align: right;
-    }
-  }
-
-  code, pre, tt {
-    font-family: $font-console;
```

Comment:
Note: removing this means we now use the same font for normal text as for text in code blocks. I think this looks fine, but I'm happy to restore this rule!

Reply:
Github seems to be using monospace, so I would suggest we do so as well.